api/Document.json

```diff
@@ -456,9 +456,16 @@
             },
             "opera": "mirror",
             "opera_android": "mirror",
-            "safari": {
-              "version_added": "12"
-            },
+            "safari": [
+              {
+                "version_added": "13.1"
+              },
+              {
+                "version_added": "12",
+                "partial_implementation": true,
+                "notes": "The event handler is exposed but will not actually be called unless both the \"Web Animations\" and \"CSS Animations via Web Animations\" preferences are enabled."
```

Comment:
For this wording, can you follow the example of `onmessageerror`? We don't need to mention the flags I think.